### ISSUE TITLE
Fails read env from file

### ISSUE THREAD

**Issue by ad-m** [mandeep/Travis-Encrypt#6]
Is there any way to load .env from file?  Something like:
```console
$ cat .env | travis-encrypt --env watchdogpolska watchdog-id .travis.yml
$ travis-encrypt watchdogpolska watchdog-id --env .travis.yml <(cat .env)
```
It fails unfortunately, He still asks a question about entering a password, although he should read it from the input.

**Comment by mandeep** [comment]
Currently, there is no way to read an environment variable from a file. I believe the issue you are seeing is with the pipes is due to the click command line library.

I'll see about adding this feature in a future release.

**Comment by mandeep** [comment]
@ad-m If you use the --password flag and supply it with something, you won't be prompted for a password. You can try --password=<(cat .env) and it may do what you want.

**Comment by ad-m** [comment]
At the moment the most useful for me is the following usage:
```
cat .travis.env | xargs -n1 travis-encrypt USER REPO --password | grep 'secure'
```

**Comment by mandeep** [comment]
Thanks for following up. Hopefully I can get to this feature soon so that reading from a file is easier.

**Comment by mandeep** [comment]
Closed by #11
